### ISSUE TITLE
Unselecting/Unhighlighting communities is unintuitive.

### ISSUE THREAD

**Issue by abueide** [vector-im/riot-web#10398]
One of the major issues I ran into that led to bad/frustrating user experience was when I made a community (the group of rooms) I couldn't figure out how to unselect it so it would show all my rooms and people. My first instinct was to click the already highlighted community icon again to unselect it, only after a bunch of fidgeting did I find out to unselect it you have to click on the empty space in the side bar.

My recommendation would be to create a "home" icon like discord has at the top left separated by a horizontal bar to indicate that it isn't a community on its own, and/or make it so when you click a highlighted community icon, it unselects it. The home icon would basically do the same thing, but just be a visual gesture for the end user so its more intuitive.

**Comment by jryans** [comment]
Yes, thanks for the feedback. It's well known that current communities is quite bad in this respect. 😭 

This seems like a duplicate of #7953, so let's track it there.
